### PR TITLE
docs: FAQ content WIP

### PR DESCRIPTION
**Pull request by misterbisson**

### CODE REVIEW

File: layouts/partials/sidebar/docs-menu.html [hunk 1]
```diff
@@ -5,20 +5,24 @@
...
 {{ define "dokschildren" }}
   {{ range $.Page.Pages }}
-    {{ $active := (eq $.currentPage.Permalink .Permalink) }}
-    <ul class="small">
-      <li class="{{ if $active }}active{{ end }}">
-        <a class="docs-link rounded{{ if $active }} active{{ end }}" href="{{ .Permalink }}">{{ .Title }}</a>
-        {{ template "dokschildren" dict "Page" . "currentPage" $.currentPage }}
-      </li>
-    </ul>
+    {{ if or (not (isset .Params "toc")) (not (eq .Params.toc false)) }}
```
Comment (misterbisson) [inline layouts/partials/sidebar/docs-menu.html:18]: Support for suppressing pages from the big sidebar table of contents on every page.

File: layouts/partials/sidebar/docs-menu.html [hunk 2]
```diff
@@ -5,20 +5,24 @@
       {{ $active := (eq $currentPage.Permalink .Permalink) }}
         <li class="mb-1">
           <h3 class="h6 text-uppercase"><a class="docs-link rounded{{ if $active }} active{{ end }}" href="{{ .Permalink }}">{{ .Title }}</a></h3>
-          {{ template "dokschildren" dict "Page" . "currentPage" $currentPage }}
+          {{ if or (not (isset .Params "toc")) (not (eq .Params.toc false)) }}
```
Comment (misterbisson) [inline layouts/partials/sidebar/docs-menu.html:8]: Support for suppressing pages from the big sidebar table of contents on every page.

File: layouts/partials/main/pages.html [hunk 3]
```diff
@@ -3,13 +3,13 @@
...
           <a class="docs-link rounded" href="{{ .Permalink }}">{{ .Title }}</a>
-          {{ template "dokschildren" dict "Page" . }}
+          {{ template "dokschildrenmain" dict "Page" . }}
         </li>
     {{ end }}
 </ul>
 {{ end }}
 
-{{ define "dokschildren" }}
+{{ define "dokschildrenmain" }}
```
Comment (misterbisson) [inline layouts/partials/main/pages.html:12]: Differentiating this template from a similarly named one elsewhere.

This template intentionally does not support suppressing pages from this table of contents.

File: content/how-to/iac-checks/gitlab/index.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ lead: "Configuring infrastructure as code on GitLab"
 date: 2022-01-12T02:48:57+00:00
 draft: false
 weight: 200
-toc: true
+toc: false
```
Comment (misterbisson) [inline content/how-to/iac-checks/gitlab/index.md:8]: Setting this across multiple pages so they don't appear in the table of contents.

File: content/faq/why-should-i-use-blubracket-discover-secrets/index.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+title: "An example question"
```
Comment (misterbisson) [inline content/faq/why-should-i-use-blubracket-discover-secrets/index.md:2]: This and many pages are stubbed out based on the list of FAQs elsewhere. The title and content need to be set.

**Review by robbyandrews — APPROVED** [review]

**Review by robbyandrews — APPROVED** [review]

**Pull request merged**